internal/gitea/gitea.go

```diff
@@ -0,0 +1,14 @@
+package gitea
+
+type RepoConfig struct {
```

Comment:
Go style nit: unlike Python, Go doesn't need things defined before they're referenced, so I'd keep `OrgConfig` at the top and define this (less important / dependent) type below it.